meetings/organize.html

```diff
@@ -0,0 +1,97 @@
+	font-weight: bold;
+	content: "Issue:";
+	}
+
+	.subtopic {
+	font-weight: bold;
+	}
+	.subtopic::after {
+	content: " - ";
+	}
+      </style>
+</head>
+
+<body>
+      <div id="header">
+            <span class="logo"><a href="/"><img src="/Icons/WWW/w3c_home_nb" alt="W3C" height="48"
+                              width="72" /></a></span>
+            <div class="breadcrumb">
+                  <a href="/Member/">Member</a> &rarr;&nbsp;<a href="/Guide/">The Art of Consensus</a> &rarr;&nbsp;
+                  <h1>Organizing a distributed meeting</h1>
+            </div>
+            <p class="baseline">This <strong>Guidebook</strong> is the collected wisdom of the W3C Group Chairs and
+                  other collaborators.</p>
+      </div>
+      <div class="toc">
+            <h4>On This Page&nbsp;&rarr;&nbsp;</h4>
+            <ul>
+                  <li><a href="#timezones">Timezones</a>&nbsp;&bull;&nbsp;</li>
+                  <li><a href="#calendar">Group Calendars</a>&nbsp;&bull;&nbsp;</li>
+            </ul>
+      </div>
+
+      <p>
+        W3C benefits from diverse, inclusive, and worldwide participation. This document provides guidance
+        to those organizing distributed meetings.
+      </p>
+
+      <h2 id='timezones'>Dealing with timezones</h2>
+
+      <blockquote cite='https://www.w3.org/Consortium/Process/#distributed-meeting'>
+            <p>A distributed meeting is one where most of the attendees are expected to participate from remote locations
+               (e.g., by telephone, video conferencing, or IRC).</p>
+            <p>&mdash; <a href='https://www.w3.org/Consortium/Process/#distributed-meeting'>W3C Process Document</a></p>
+      </blockquote>
+      <p>A chair should build consensus about the time slots for distributed meetings and should
+	 reevaluate that consensus on a regular basis, e.g. every 6 months.</p>
+
+      <p>A chair might use the following steps for proposing time slots:
+      <ul>
+        <li>Collect participants' time zones</li>
+        <li>Propose candidate time slots, at least some of which start beween 6:00am and 12:00am in each participant's timezone
+                (see <a href="https://www.timeanddate.com/worldclock/meetingtime.html?iso=20210127&p1=240&p2=248&p3=33&p4=195&p5=136&p6=43&p7=64&p8=224">World Clock Meeting Planner</a>)
+        </li>
+        <li>If using a tool such as doodle.com to collect feedback, provide alternative mechanisms to participants who
+	have trouble accessing the tool</li>
+      </ul>
+      <p>A chair should consider the following when judging the group's consensus:</p>
+      <ul>
+        <li>Prioritize preferences of core participants based on the meeting agenda</li>
+        <li>Prioritize the major preference of the other members but should take account that a
+                  preference of the majority in a time zone area has an equivalent priority to
+                  those of the majority in other time zone areas.</li>
```

Comment:
I completely agree.  This is [the bullet item I couldn't parse](https://github.com/w3c/Guide/pull/112#issuecomment-769158287), so I didn't [rewrite it](https://github.com/w3c/Guide/pull/113).

Reply:
I wrote the sentence to mitigate the problem that a time of a regularly distributed meeting is simply determined depending on the major preference without taking into account the preferred time zones of participants.  For example, when only a few participants are from Asia but the most participants are from the US and Europe, the US and Europe friendly time is always selected. I think that in such a case the preferred time of majority in Asia should be taken into account for equality

Any suggestion to modify the sentence is welcome. Any other sentence to mitigate the problem is also welcome.

Reply:
Are you saying that it's *not* equitable if the time preference of the majority is always chosen?

Reply:
How about:

"Balance the majority preference of the other participants with preferences from the minority participants. Favoring the timezone of choice from the majority for a regular distributed meeting would systematically disenfranchise the participants in less favorable timezones."

Reply:
"consistently favoring" (i.e. if 2/3rds of the group comes from one timezone, maybe it's OK if 2/3rds of the meetings favor that, but not if they all do)

Reply:
> Are you saying that it's _not_ equitable if the time preference of the majority is always chosen?

Yes, I am.  It is the point that I want to get a consensus in W3C.  We can insist that it is not equitable but it is hard for us to agree on what is equitable.  If my sentence is not good, I am OK with the sentence that @plehegar suggested.

Reply:
> "consistently favoring" (i.e. if 2/3rds of the group comes from one timezone, maybe it's OK if 2/3rds of the meetings favor that, but not if they all do)

I think that rather than defining a strict guideline, we should give it to a char's description to make a consensus on balancing preferences of timezone in the group.  I think an easier rule just to facilitate balancing timezone would be sufficient.

I do not think just balancing the meeting timezone is not always better. For example, if only one person is from one timezone but they is a core participant, the char may prioritize their preference of timezone with a consensus in the group.  The following is also a best practice of chairs.

> Prioritize preferences of core participants based on the meeting agenda

Reply:
I agree with @igarashi50 - just choosing the time preference of the majority is not an equitable solution.

Unfortunately it often happens that chairs do that because those in the majority just skip meetings not scheduled when convenient to them, making such meetings ineffective. That's an indictment on our commitment to equity and inclusion.

Reply:
The point regarding core participants is in the previous item. No need to bring it up again in the paragraph.
The list start with "A chair should consider the following when judging the group's consensus" so this is a list of consideration for the Chair. So, the whole set of considerations is the balancing act.

Reply:
I don't like the word "balance" in this context.  How about instead:

"Give consideration to all preferences, including those in the minority, taking care to not systematically disenfranchise any participants."

Reply:
I can go with this tweak.